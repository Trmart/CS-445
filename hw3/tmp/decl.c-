## C-S23
main()
 begin 
   int a;
   bool b;
   char c;
 end 
