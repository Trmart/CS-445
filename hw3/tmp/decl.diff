3,4c3,5
< Func: main returns type void [line: 2]
< Number of errors: 0
---
> ERROR(LINKER): A function named 'main()' must be defined.
> Func: ( returns type void [line: 2]
> Number of errors: 1
