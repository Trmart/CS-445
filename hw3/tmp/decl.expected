.   .   Child: 0  Var: a of type int [line: 4]
.   Child: 1  Compound [line: 3]
Func: main returns type void [line: 2]
Number of errors: 0
Number of warnings: 3
.   .   Sibling: 1  Var: b of type bool [line: 5]
.   .   Sibling: 2  Var: c of type char [line: 6]
WARNING(4): The variable 'a' seems not to be used.
WARNING(5): The variable 'b' seems not to be used.
WARNING(6): The variable 'c' seems not to be used.
